QUARKUS-2743.md

```diff
@@ -0,0 +1,59 @@
+# QUARKUS-2743 - Consuming multiple media types from a resource method does not work
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-2743
```

Comment:
Up, noticed this too late. 

These 4 lines are rendered on on line. Either add "-" in front of each line or new empty line between each line

@michalvavrik pls create new PR to fix this

Reply:
Sure, not the first time I made this mistake; https://github.com/quarkus-qe/quarkus-test-plans/pull/122